### PR TITLE
Add lime3ds and deprecate citra

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- lime3ds: Add at v2114.1
- citra : Deprecate
- repo_data : Deprecate citra

**Test Plan**

Play some games

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Comment by malfisya** [comment]
@infinitymdm Can you please test this too? Also want to co-maintain it?

File: packages/l/lime3ds/pspec_x86_64.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            <Path fileType="header">/usr/include/enet/enet.h</Path>
+            <Path fileType="header">/usr/include/enet/list.h</Path>
+            <Path fileType="header">/usr/include/enet/protocol.h</Path>
+            <Path fileType="header">/usr/include/enet/time.h</Path>
+            <Path fileType="header">/usr/include/enet/types.h</Path>
+            <Path fileType="header">/usr/include/enet/unix.h</Path>
+            <Path fileType="header">/usr/include/enet/utility.h</Path>
+            <Path fileType="header">/usr/include/enet/win32.h</Path>
+            <Path fileType="library">/usr/lib64/libenet.a</Path>
+        </Files>
```
Comment (malfisya) [inline packages/l/lime3ds/pspec_x86_64.xml:51]: I am quite sure how to disable this. Citra doesn't have `-devel` packages and those files is certainly not lime's

**Comment by infinitymdm** [comment]
> @infinitymdm Can you please test this too? I am pretty much just copying `citra` package.yml. Also want to co-maintain it?

I'd be happy to!

File: packages/l/lime3ds/pspec_x86_64.xml (hunk 1 above)
Comment (malfisya) [reply packages/l/lime3ds/pspec_x86_64.xml]: Used system `libenet` solves this

**Review by joebonrichie — DISMISSED** [review]

**Pull request merged**